api/swagger.yaml

```diff
@@ -838,12 +838,11 @@ definitions:
       Volumes:
         description: "An object mapping mount point paths inside the container to empty objects."
         type: "object"
-        properties:
-          additionalProperties:
-            type: "object"
-            enum:
-              - {}
-            default: {}
+        additionalProperties:
+          type: "object"
+          enum:
+            - {}
+          default: {}
```

Comment:
Are these `enum` and `default` lines necessary? I think they can be removed.

Reply:
As the description say: "An object mapping mount point paths inside the container to empty objects.". I think it helps a lot understanding (and also validating) that only empty object is allowed as a value here.